Shared/Samples/Change map view background/ChangeMapViewBackgroundView.Model.swift

```diff
@@ -0,0 +1,86 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import SwiftUI
+import ArcGIS
+
+extension ChangeMapViewBackgroundView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        /// A map with a streets basemap style.
+        let map: Map = {
+            // Initialize tiled layer.
+            let tiledLayer = ArcGISTiledLayer(url: .worldTimeZones)
+            
+            // Initialize map with tiled layer as basemap.
+            let map = Map(basemap: Basemap(baseLayer: tiledLayer))
+            return map
+        }()
+        
+        /// The background grid for the map.
+        let backgroundGrid = BackgroundGrid(backgroundColor: .black, lineColor: .white, lineWidth: 2, size: 32)
+        
+        // The line width range.
+        // Used by Slider, which requires CGFloat values.
+        let lineWidthRange = CGFloat(0)...CGFloat(10)
+        
+        // The grid size range.
+        // Used by Slider, which requires CGFloat values.
+        let sizeRange = CGFloat(2)...CGFloat(50)
+        
+        /// The background color of the grid.
+        @Published var color: Color {
+            didSet {
+                backgroundGrid.backgroundColor = UIColor(color)
+            }
+        }
+        
+        /// The color of the grid lines.
+        @Published var lineColor: Color {
+            didSet {
+                backgroundGrid.lineColor = UIColor(lineColor)
+            }
+        }
+        
+        /// The width of the grid lines in device-independent pixels (DIP).
+        @Published var lineWidth: CGFloat {
+            didSet {
+                backgroundGrid.lineWidth = lineWidth
+            }
+        }
+        
+        /// The size of each grid square in device-independent pixels (DIP).
+        @Published var size: CGFloat {
+            didSet {
+                backgroundGrid.size = size
+            }
+        }
+        
+        init() {
+            // Initialize properties from the background grid.
+            color = Color(uiColor: backgroundGrid.backgroundColor)
+            lineColor = Color(uiColor: backgroundGrid.lineColor)
+            lineWidth = backgroundGrid.lineWidth
+            size = backgroundGrid.size
+        }
+    }
+}
+
+private extension URL {
+    static let worldTimeZones = URL(
+        string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/WorldTimeZones/MapServer"
```

Comment:
sampleserver6 has some stability issues and this URL had a history of malfunctioning in the past. 

I'm a bit occupied right now, so I'd suggest searching for previous discussions, and asking in the sdk-samples channel, and see if someone can help to find a data replacement. Tag me as well, and if no one gets to it, I'll take a look later next week.

Reply:
Done. I'm using this Esri service instead: https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/World_Time_Zones/FeatureServer/0

Reply:
Thanks! Would you use the portal item https://www.arcgis.com/home/item.html?id=312cebfea2624e108e234220b04460b8 instead of the feature service URL in the code? It would make finding the data by portal ID easier for future maintenance.

Reply:
Done.